docs/config.yaml

```diff
@@ -55,6 +55,9 @@ streams:
         legacy_oscontainer_old: registry.ci.openshift.org/rhcos-devel/machine-os-content
       # OPTIONAL: override default coreos-assembler image to build with
       cosa_img: quay.io/coreos-assembler/coreos-assembler:stable
+      # OPTIONAL/TEMPORARY: override default coreos-assembler image to use for
+      # the controller pod in a multi-arch build
+      cosa_controller_img: quay.io/coreos-assembler/coreos-assembler:latest
```

Comment:
suggestion: `cosa_build_arch_job_controller_img_hack`

This maybe should have a more complete comment with more context too.